src/Resources/config/config.xml

```diff
@@ -117,4 +117,24 @@
             </helpText>
         </input-field>
     </card>
+    <card>
+        <title>Delayed Capture</title>
```

Comment:
Can we change the wording of these labels to be consistent with our [docs](https://docs.adyen.com/online-payments/capture#manual-capture)?
```suggestion
        <title>Manual Capture</title>
```